common/pop_types/cwp_engineers.txt

```diff
@@ -64,6 +64,15 @@
 					desc = QUALIFICATIONS_FAVORED_TYPE
 				}
 			}
+			if = {
+				limit = {
+					is_pop_type = investors
+				}
+				multiply = {
+					value = 3
+					desc = QUALIFICATIONS_FAVORED_TYPE
+				}
+			}
```

Comment:
yeah idk how I feel about investors being a preferred type for engineers. Most of the investors were meant to come from professionals since they represent the gilded class of workers that are more likely to move up a strata.

Reply:
investors and landlords are meant to be equivalent, why engineers would be more likely to be rural business owners rather than urban ones?

Reply:
Professionals work in urban buildings, engineers work in industry and rural buildings.  I think the distinction makes sense.

Reply:
So:
Professionals <-> investors
Engineers <-> investors and landlords
Farmers <-> landlords
investors <-> landlords
That would be okay for favored types?

Reply:
Yeah, that sounds good